### DISCUSSION TITLE
Where's micromark.js?

### DISCUSSION THREAD

**Discussion by capr** [micromark/micromark#65] [Q&A] [answered]
Hi,

Where's `micromark.js` that I can just load in the browser and use with a simple call to `micromark_parse(md_string) -> html_string`?

How do I get there? 

Thanks!

**Comment by wooorm** [comment, marked as answer]
Use a build tool like esbuild, webpack, or rollup. Or use tools like unpkg or so

**Reply by capr** [reply]
Hi, Can you be more specific? I never used those tools before.

**Reply by wooorm** [reply]
https://unpkg.com

**Reply by capr** [reply]
Really helpful, thanks.
